lld/test/ELF/x86-64-dwarf5-debug-names-type-comdat.test

```diff
@@ -0,0 +1,16 @@
+// REQUIRES: x86
```

Comment:
In newer tests we make the comment (non-RUN non-CHECK) marker outstanding. We typically use `# RUN:` `# CHECK:` and `## Regular comments.`